libs/designsystem/button/src/button.component.scss

```diff
@@ -172,6 +172,7 @@ $button-margin: utils.size('xxxs');
   user-select: none;
   vertical-align: middle;
   white-space: nowrap;
+  text-decoration: none;
```

Comment:
Consider matching this declaration with `a` elements only to make it more explicit, that these styles target hyperlink.

https://developer.mozilla.org/en-US/docs/Web/CSS/:host_function

Reply:
Good idea!